library/src/scala/forceInline.scala

```diff
@@ -0,0 +1,8 @@
+package scala
+
+/** An annotation on methods that is equivalent to Dotty `inline` modifier.
+ *
+ *  The annotation should be used instead of the `inline` modifier in code
+ *  that needs to cross compile between Scala 2 and Dotty.
```

Comment:
You should also mention that `@forceInline` will be ignored by Scala 2. Maybe hint at the need of `@forceInline @inline` for cross compilation with the optimization in both versions.